src/birthder.js

```diff
@@ -209,12 +243,35 @@
     return message
   }
 
+  /**
+   * Check if there is a birthday yesterday
```

Comment:
Find all the birthdays which were yesterday and remove the channels created for them.